<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;gfgg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="add_vyrobce">
      <attribute name="title">
       <string>Přidat Výrobce</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="vyrobce">
         <property name="text">
          <string>Název Výrobce</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nazev_Vyrobce"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="add_pilu">
      <attribute name="title">
       <string>Přidat Pilu</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QSpinBox" name="addVykon"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="delka_listy">
         <property name="text">
          <string>Delka Lišty</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="addDelkaListy"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nazevPily">
         <property name="text">
          <string>Nazev Pily</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addNazevPily"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="vykon">
         <property name="text">
          <string>Vykon</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="add_prodejnu">
      <attribute name="title">
       <string>Přidat Prodejnu</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="nazevProdejny">
         <property name="text">
          <string>Název Prodejny</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addNazev"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mesto">
         <property name="text">
          <string>Město</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="addMesto"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ulice">
         <property name="text">
          <string>Ulice</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="psc">
         <property name="text">
          <string>PSC</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="tel">
         <property name="text">
          <string>TEL</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="addUlice"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="addPSC"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="addTEL"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
